.github/workflows/release_subctl_devel.yml

```diff
@@ -19,24 +19,19 @@ jobs:
 
       - name: Generate subctl version
         run: |
-          echo "SUBCTL_TAG=${GITHUB_REF##*/}" >> $GITHUB_ENV
-          echo "RELEASE=release-${GITHUB_REF##*/}" >> $GITHUB_ENV
+          echo "BRANCH=${GITHUB_REF##*/}" >> $GITHUB_ENV
+          echo "RELEASE=subctl-${GITHUB_REF##*/}" >> $GITHUB_ENV
 
       - name: Generate the subctl release artifacts
-        run: make build-cross VERSION=${{ env.SUBCTL_TAG }}
+        run: make build-cross VERSION=${{ env.BRANCH }}
 
-      - name: Install GH CLI
-        run: |
-          curl -sLo /tmp/gh.deb https://github.com/cli/cli/releases/download/v1.11.0/gh_1.11.0_linux_amd64.deb
-          sudo apt install /tmp/gh.deb
-
-      - name: Delete old release
-        run: git push -d origin ${{ env.RELEASE }} || true
-
-      - name: Create the release with the updated artifacts
+      - name: Recreate the release with the updated artifacts
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          gh release delete ${{ env.RELEASE }} -y || true
+          git push -d origin ${{ env.RELEASE }} || true
           gh release create ${{ env.RELEASE }} dist/*.tar.xz --prerelease \
-            --title "Cutting Edge: ${{ env.SUBCTL_TAG }}" \
+            --title "Cutting Edge: ${{ env.BRANCH }}" \
```

Comment:
but binaries need to go to subctl-devel or subctl-release-0.9 otherwise we need to change the scripts. Won't this mess with the release-xxx or devel branch?

Reply:
this is just the name of the release, not sure what youre thinking about

Reply:
Oh right, this is the title.. RELEASE=subctl-xxxx 

not feeling very well today :-P :-/